### DISCUSSION TITLE
Can we get a 'First Fourth' Top and Bottom option?

### DISCUSSION THREAD

**Discussion by BanAaron** [rxhanson/Rectangle#533] [Ideas]
What I am trying to achieve is this setup:

<img width="3840" alt="Screenshot 2021-08-05 at 5 39 20 pm" src="https://user-images.githubusercontent.com/4992248/128388617-670f734d-1e67-49a6-9a6c-4b62bf91eb49.png">

Currently what I have to do is set one window to Center Half and set the side ones to First Fourth and manually adjust the window heights. It would be great if we just set First Fourth Top and Bottom and the same on Last Fourth

I understand this is probably quite neice because it only really makes sense on 32:9 monitors, but it would be the iceing on the cake for me ❤️

**Comment by rxhanson** [comment]
Thanks for the feedback!

In an upcoming Rectangle release I am planning to make it easier for other developers to add additional dimension shortcuts. 

This is actually really easy to achieve with the custom shortcut builder in my paid superset of Rectangle, [Hookshot](https://hookshot.app), and here's a blog post about it: https://medium.com/ryan-hanson/custom-window-dimension-shortcuts-in-macos-e7df347dfd24
